.gitignore

```diff
@@ -1 +1,2 @@
-/node_modules
\ No newline at end of file
+/node_modules
+.env
```

Comment:
please add .env.sample file in our project

Reply:
ok